integration_tests/expr_05.py

```diff
@@ -35,19 +38,24 @@ def main0():
     assert -105346 ^ -32771 == 72579
     assert 10 >> 1 == 5
     assert 5 << 1 == 10
-    i1: i32 = 10
-    i2: i32 = 4
-    assert i1 << i2 == 160
-    assert i1 >> i2 == 0
```

Comment:
This does not work?

Reply:
Everything works, I just moved the tests below.

Reply:
I just tested it manually, it seems to work. Let's keep this test there, I want to make sure we do not break anything accidentally.

Reply:
Fixed in f169f16bc6645dfc6bff2c0f5b80b9e4785ba9f2.

Reply:
I see -- you moved the tests down. I restored the original order, just to make it easy to review, to see that nothing was lost by a mistake (see my updated fix in 0dffbd6fe7fdbd3810f1ce1d544b51dadb7343b6).

If you like the new order better, let's submit a new PR after this is merged.